tests/many_to_one/tests.py

```diff
@@ -576,6 +577,15 @@ def test_fk_to_bigautofield(self):
         District.objects.create(city=ny, name='Brooklyn')
         District.objects.create(city=ny, name='Manhattan')
 
+    def test_fk_to_smallautofield(self):
+        us = Country.objects.create(name='United States')
+        City.objects.create(country=us, name='Chicago')
+        City.objects.create(country=us, name='New York')
+
+        uk = Country.objects.create(name='United Kingdom', id=2 ** 11)
```

Comment:
Why `2**11 = 2048` and not the upper bound, i.e. `2**15 - 1`?

Reply:
I just took a value expected to fit rather than the upper bound. Very much the same way as `BigAutoField` using `2 ** 33` instead of `2 ** 63 - 1`. We should probably tweak that in a separate commit too then.

Reply:
I was just curious why we chose them, it seems that we want to test creation with manually specified `ID`. It can stay.